### PR TITLE
change angular material to materializa

### PR DESCRIPTION
**Pull request by fakhrihawari**
angularjs material component that changed using materializecss and css :
-md-select
-md-select with search header
-chips
-md-tooltip change

### CODE REVIEW

File: app/scripts/app/directives/materialize.select.js [hunk 1]
```diff
@@ -26,7 +26,9 @@ angular.module("ngm.materialize.select", [])
 
 																// if searchable then prepend search
 																if (attrs.searchable) {
-																	addSearch();
+                                                                    setTimeout(() => {
```
Comment (drfaustusfade) [inline app/scripts/app/directives/materialize.select.js:29]: remove $('select').formSelect(); on details html, and no need for timeout here, it's better not to use page formSelect

**Comment by drfaustusfade** [comment]
can we comment angular material on index.htm?

File: app/scripts/app/directives/materialize.select.js (hunk 1 above)
Comment (fakhrihawari) [reply app/scripts/app/directives/materialize.select.js]: updated

**Comment by fakhrihawari** [comment]
> can we comment angular material on index.htm?

maybe later i still not try it

**Pull request merged**